src/otx/algorithms/common/adapters/mmcv/utils/automatic_bs.py

```diff
@@ -126,6 +127,10 @@ def _set_batch_size(cfg, batch_size: int):
         cfg.data.videos_per_gpu = batch_size
     else:
         cfg.data.train_dataloader["samples_per_gpu"] = batch_size
+        update_or_add_custom_hook(cfg, {"type": "AdaptiveRepeatDataHook", "train_batch_size": batch_size})
```

Comment:
Are there any reasons why `AdaptiveRepeatDataHook` is added? Isn't is sufficient to change batch size value if `AdaptiveRepeatDataHook` exists?

Reply:
I forgot the exact reason, however, if we didn't include the `AdaptieRepeatDataHook`, it raised an error at some cases.

Reply:
If auto_bs is enabled (i-seg, s-seg), the batch size is changed by automatic_bs. So, the batch size of the `AdaptiveRepeatDataHook` also should be changed. That's the reason why I added this line.

Reply:
Changed.